pangea-sdk/service/redact/api.go

```diff
@@ -88,6 +88,9 @@ type TextInput struct {
 
 	// If the response should include some debug Info.
 	Debug *bool `json:"debug,omitempty"`
+
+	// An array of redact ruleset short names
+	Rules *[]string `json:"rules,omitempty"`
```

Comment:
Should not be a pointer. `[]string can be nil, and omitempty will work`

Reply:
Fixed!